modules/social_features/social_core/social_core.install

```diff
@@ -892,3 +893,11 @@ function social_core_update_8501() {
   $config->save();
 
 }
+
+/**
+ * Allow using the "Mail HTML" text format by the logged user.
+ */
+function social_core_update_8601() {
+  // Set some default permissions.
+  _social_core_set_permissions();
```

Comment:
This could potentially enable more permissions if distro users wanted to disable them through the UI they will be enabled again. Next time just singularly add one permission that would be better :).

Reply:
Done.